src/components/Fieldset/Fieldset.module.css

```diff
@@ -10,5 +10,6 @@
  */
 .fieldset {
   @mixin fieldsetStyles; /* 1 */
+  --checkbox-svg-size: normal;
```

Comment:
Removes some extra margin bottom on the checkbox. Scoping it here since changing it on the Checkbox component will affect Checkbox sizes across LP

Reply:
Ah great question, turns out I had to move where the style was being placed in `Checkbox`, thanks for the catch